frontend/server/src/Controllers/User.php

```diff
@@ -7,6 +7,7 @@
  *
  * @psalm-type PageItem=array{class: string, label: string, page: int, url?: string}
  * @psalm-type AuthorsRank=array{ranking: list<array{author_ranking: int|null, author_score: float, classname: string, country_id: null|string, name: null|string, username: string}>, total: int}
+ * @psalm-type AuthorsRankWithQualityProblems=array{ranking: list<array{user_id: int|null, author_ranking: int|null, name: null|string, username: string}>}
```

Comment:
```suggestion
 * @psalm-type AuthorsRankWithQualityProblems=array{ranking: list<array{user_id: int|null, author_ranking: int, name: null|string, username: string}>}
```

hay manera de hacer que el ranking no sea nulo?